app/client/views/residents/residents.js

```diff
@@ -6,6 +6,8 @@ Template.residents.onCreated(function() {
 
   // Subscribe to all homes, for data table
   templateInstance.subscribe("allHomes");
+  templateInstance.subscribe("allResidents");
```

Comment:
I would prefer we didn't subscribe to all residents, so we only send necessary data to the client when possible. We have some views that are slow to render, due to the amount of data on the wire as our deployment grows.

Rather, consider moving the subscription to the table cell where the resident name is rendered. Add a template-level subscription to the "single resident" publication, passing in the `residentId` from the current row.

We will ne exploring server-side pagination and global client state management (e.g. Vuex) in subsequent work, where we can further optimize our views.

Reply:
It is being used in enroll uses, password change and add new feeling, and it is working fine.